heralding/honeypot.py

```
@@ -59,13 +60,26 @@ def __init__(self, config, loop):
                 logger.warning('Could not request public ip from ipify, error: %s', ex)
             await asyncio.sleep(3600)
 
+    def setup_wordlist(self):
+        # load wordlist in memory
+        try:
+            wordlist_file = self.config['hash_cracker']['wordlist_file']
+            with open(wordlist_file, 'r') as f:
+                Honeypot.wordlist = f.read().splitlines()
+        except FileNotFoundError:
+            logger.error("Error: "+wordlist_file+" file not found in path")
```

This is a critical error, we should fail hard.